web/main.go

```diff
@@ -4,12 +4,11 @@ import (
 	"context"
 	"net/http"
 	"os"
+
 	"github.com/apex/log"
 	jsonloghandler "github.com/apex/log/handlers/json"
 	"github.com/apex/log/handlers/text"
-	// This is a hardcoded variable <-- should be moved
 	"github.com/unee-t-ins/invite"
-	// END This is a hardcoded variable
```

Comment:
Best to leave this comment: it will help us in the future to list all the hardcoded variables that we need to be mindful of.